package.json

```diff
@@ -22,32 +21,30 @@
     "url": "git://github.com/goodeggs/goodeggs-test-helpers.git"
   },
   "dependencies": {
-    "bluebird": "^3.3.4",
-    "chai": "^3.3.0",
-    "chai-as-promised": "^5.1.0",
-    "chai-datetime": "^1.4.1",
-    "chai-json-schema": "^1.4.0",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "chai-datetime": "^1.5.0",
     "chaid": "^1.0.2",
-    "dirty-chai": "^1.2.2",
-    "lodash.isfunction": "^3.0.8",
-    "sinon": "^4.1.6",
-    "sinon-chai": "^2.7.0"
+    "dirty-chai": "^2.0.1",
+    "istanbul": "^0.4.5",
```

Comment:
IIRC I couldn't get that version of istanbul to work properly with babel, which is why [we were using an alpha version](https://github.com/goodeggs/goodeggs-ops-api-test-helpers/blob/ed40fc211563a8bd6fb24ec42ac2eaa509805af7/package.json#L25). Did you try running tests with `--coverage`?

Reply:
@serhalp do you remember what wouldn't work? Seems to work as expected with that 0.4.5.

Reply:
It just... didn't work at all with babel. If you tried it with `--coverage` and it worked, that's good enough for me. :+1: